.github/workflows/CI_actions.yml

```
@@ -1,5 +1,6 @@
 # This workflow will setup GitHub-hosted runners and install the required dependencies for viziphant documentation.
-# On a pull requests and on pushes to master it will build and create the documentation .
+# On pull requests and on pushes to master it will build the documentation .
+
 
 name: Viziphant-tests
```

This name appears on the badge, maybe its better to just call it "tests":
```suggestion
name: tests
```

---

agree